run-tests.php

```diff
@@ -1861,8 +1861,11 @@ function run_test($php, $file, $env)
 			}
 
 		} else {
-
-			if (!isset($section_text['PHPDBG']) && @count($section_text['FILE']) + @count($section_text['FILEEOF']) + @count($section_text['FILE_EXTERNAL']) != 1) {
```

Comment:
Personally I'd just replace these `@count()` with `isset()`.

Reply:
Replacing with ``isset()`` would change the logic of this condition as the usage of ``count()`` here is to assert that ONLY one of the sections is present otherwise the test file is bogus, same below.

Reply:
To be clear, this is what I had in mind:

```suggestion
			if (!isset($section_text['PHPDBG']) && isset($section_text['FILE']) + isset($section_text['FILEEOF']) + isset($section_text['FILE_EXTERNAL']) != 1) {
```

Too dirty?

Reply:
Hum, the intention of this is kinda lost, I'll probably have a look at it again later and try to see if I can't refactor the codebase a bit better.